File: Practica 1/Punto.py
import math
class Punto:
    def _init_(self, x, y):
        self.x = x
        self.y = y

    def coord_cartesianas(self):
        print(f"Coordenadas Cartesianas: ({self.x}, {self.y})")

    def coord_polares(self):
        r = (self.x*2 + self.y2) * 0.5
        theta = math.atan2(self.y, self.x)
        print(f"Coordenadas Polares: (r = {r}, θ = {theta})")

    def _str_(self):
        return f"Punto(x={self.x}, y={self.y})"

File: Practica 1/Punto.java
public class Punto {
    public float x;
    public float y;

    public Punto(float x, float y) {
        this.x = x;
        this.y = y;
    }

    public void coord_cartesianas() {
        System.out.println("Coordenadas Cartesianas: (" + this.x + ", " + this.y + ")");
    }

    public void coord_polares() {
        double r = Math.sqrt(x * x + y * y);
        double theta = Math.atan2(y, x);
        System.out.println("Coordenadas Polares: (r = " + r + ", θ = " + theta + ")");
    }

    @Override
    public String toString() {
        return "Punto(" + "x=" + x + ", y=" + y + ')';
    }

    public static void main(String[] args) {
        Punto punto = new Punto(3, 4);
        punto.coord_cartesianas();
        punto.coord_polares();
        System.out.println(punto);
    }
}